src/Docker/DatabaseServices.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Docker;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ *
+ * @internal
+ */
+final class DatabaseServices
+{
+    public static function getDatabaseSkeleton(string $name, string $version): array
+    {
+        switch (strtolower($name)) {
```

Comment:
Could we remove the `strtolower`?

Reply:
I agree, AFAIK `^[a-zA-Z0-9._-]+$` this is a validation pattern for this name. I didn't found anything in the docker documentation but I found this specs, I think it comes from YAML as a valid key. https://github.com/compose-spec/compose-spec/blob/master/schema/config_schema_v3.9.json

Reply:
i just realized case is doing == not === so I can get rid of those here and below..

Reply:
done